ocs_ci/ocs/ui/views.py

```diff
@@ -1708,3 +1708,9 @@
         "validation": validation,
     },
 }
+
+
+locate_aws_regions = {"region_table": ('//*[@id="w101aac10b9c15"]', By.XPATH)}
```

Comment:
Looks like a dynamic id used in xpath and highly likely to change in each run, pls check